Week-1/week1_assignment_1_2_cattyBoard/cattyBoard.html

```diff
@@ -0,0 +1,197 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <style>
+    * {
+      box-sizing: border-box;
+    }
+
+    header {
+      float: left;
+      display: flex;
+      background-color: #3a3b3c;
+      padding-left: 25px;
+      width: 100vw;
+      text-align: left;
+
+      color: white;
+    }
+
+    nav {
+      position: absolute;
+      float: left;
+      width: 3em;
+      height:85vh;
+      background: #a9a9a9;
+      padding: 10px;
+    }
+    li button {
+
+      border: none;
+      outline: none;
+      background: none;
+
+    }
+    p button {
+      padding-left:2em;
```

Comment:
Please use the "multiple of 4" principle here.
Also... it is better to use "rem" instead of "em"
and never ever use "em" in lists... (just fyi)

"em" takes its direct parent as reference... It can get out of control easily. 
"rem" is safer! as it takes the root "html" element as its reference.